README.md

```diff
@@ -451,6 +452,27 @@ Scala, Python, Spark, PySpark, Open Source#### Streaming on:
 - [GitHub](https://github.com/keraf)
 - [Website](https://ker.af/)
 
+### Kevin Griffin
+#### What Kevin Streams
+- C#, .NET, ASP.NET, 3D Printing, Conference Planning, and more!
+#### Streaming on:
+- [Twitch](https://www.twitch.tv/1kevgriff)
+#### Links:
+- [Twitter](https://twitter.com/1kevgriff)
+- [GitHub](https://github.com/1kevgriff)
+- [Website](https://kevgriffin.com/)
+
+### KeyFramers
```

Comment:
It looks like you duplicated the KeyFramers section. Please remove it.

Reply:
Interesting - not duped in my fork.  I'll review - sorry!